src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/events/CiagInductionTimelineEventTest.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service.events
+
+import org.awaitility.kotlin.await
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.verify
+import software.amazon.awssdk.services.sns.model.MessageAttributeValue
+import software.amazon.awssdk.services.sns.model.PublishRequest
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.aValidPrisonNumber
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.TimelineEventType
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.assertThat
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.service.events.aValidCiagInductionCreatedEvent
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.service.events.aValidCiagInductionUpdatedEvent
+
+internal class CiagInductionTimelineEventTest : IntegrationTestBase() {
+
+  @Test
+  fun `should process CIAG induction created event`() {
+    // Given
+    val prisonNumber = aValidPrisonNumber()
+    val inductionCreatedEvent = aValidCiagInductionCreatedEvent(prisonNumber = prisonNumber)
+
+    // When
+    inboundSnsClient.publish(
+      PublishRequest.builder()
+        .topicArn(inboundTopicArn)
+        .message(objectMapper.writeValueAsString(inductionCreatedEvent))
```

Comment:
This sucks that we have to do everything with strings of json 😢 
Surely these classes can be typed (I know it was different, but we had strongly typed messages in VCA. It was all still AWS' SDKs so I'd be surprised if it didnt support it 🤔 )